draft-ietf-opsawg-pcapng.md

```diff
@@ -2220,6 +2228,182 @@ The following is a list of Secrets Types.
+location, orientation, velocity, and related errors.
+Each CIB is associated with a specific interface and applied to all packets
+received on this interface following the CIB until superseded by a new CIB on
+the same interface.
+
+For example, a capture from a static interface may contain a single CIB at
+the start of the file to apply to all packets on this interface.
+A capture from a moving interface with GNSS will add a CIB each time updated
+positional information is available, and this information will apply to all
+packets received on this interface until the next CIB.
+
+Note that issuing a new CIB will clear all existing CIB options on an interface,
+for example, the CIBs for a non-moving but rotating interface with a known position
+must all contain both location and orientation data.
+
+The format of the Capture Information Block is shown in {{format_cib}}.
+
+
+~~~~
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +---------------------------------------------------------------+
+ 0 |                   Block Type = 0x0000000B                     |
+   +---------------------------------------------------------------+
+ 4 |                      Block Total Length                       |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ 8 |                         Interface ID                          |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+12 /                                                               /
+   /                      Options (variable)                       /
+   /                                                               /
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                      Block Total Length                       |
+   +---------------------------------------------------------------+
+~~~~
+{: #format_cib title='Capture Information Block Format' artwork-align="center"}
+
+The fields have the following meaning:
+
+* Block Type: The block type of the Capture Information Block is 11.
+
+* Block Total Length: total size of this block, as described in
+  {{section_block}}.
+
+* Interface ID: specifies the interface for this CIB to be applied
+  to; the correct interface will be the one whose Interface
+  Description Block (within the current Section of the file) is
+  identified by the same number (see {{section_idb}}) of this field.
+  The interface ID MUST be valid, which means that an matching
+  interface description block MUST exist.
+
+* Options: optionally, a list of options (formatted according to
+  the rules defined in {{section_opt}}) can be present.
+
+
+Location information can be represented with in global (lat/lng/alt in degrees)
+or local (X/Y/Z in metres) form, only one of these forms may be used in a given PCAP file.
+Combining pcap files with different location formats should result in the
```

Comment:
I would like some imput regarding this one. If both Global and Local location information is used in the same pcap files it should result in the removal of all position information: Would it make sense to include a global location information to tag the specific pcap in a global position and then start using local coordinates referencing this origin? Does this make sense for any usecase? Thanks

Reply:
Would it make sense to use global + local positioning in a case where you are performing a capture in a reference frame that is moving on which you would like to have local measurements of high precission? Example: Performing captures on a train where local frame would map the interior of the train but global one would position the train globally? 🤔